website/docs/r/pi_instance.html.markdown

```diff
@@ -62,7 +64,8 @@ Review the argument references that you can specify for your resource.
 - `pi_key_pair_name` - (Required, String) The name of the SSH key that you want to use to access your Power Systems Virtual Server instance. The SSH key must be uploaded to IBM Cloud.
 - `pi_memory` - (Required, Float) The amount of memory that you want to assign to your instance in gigabytes.
 - `pi_migratable`- (Optional, Bool) Indicates the VM is migrated or not.
-- `pi_network_ids` - (Required, String) The list of network IDs that you want to assign to the instance. 
+- `pi_network_ids` - (Required, List of String) The list of network IDs that you want to assign to the instance. This attribute is **Deprecated** use `pi_network` instead.
+- `pi_network` - (Required, List of Map) List of one or more networks to attach to the instance. The pi_network object structure is documented below. 
```

Comment:
The pi_network object structure is documented below. 
as

must be in code tag `pi_network`

Reply:
done